cfn-resources/teams/Makefile

```diff
@@ -17,3 +17,6 @@ debug:
 
 clean:
 	rm -rf bin
+
+unit:
```

Comment:
are these tests being executed in CI? Would seem a slight adjustment in https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/Makefile#L44 is needed. (If change is not trivial we can handle in separate PR)

Reply:
I actually realised that adding 

```
//go:build unit
```

on top of the `team_user_test.go`

I see this warning message on the `package teamuser`:

```
This file is within module "cfn-resources", which is not included in your workspace.
To fix this problem, you can add a go.work file that uses this directory.
See the documentation for more information on setting up your workspace:
https://github.com/golang/tools/blob/master/gopls/doc/workspace.md.
```

So I am going to remove this change and the makefile change and do it separately in another issue.

Reply:
FYI @andreaangiolillo

Reply:
Resolving this one, please re-open if needed.